aten/src/ATen/native/cuda/TensorModeKernel.cuh

```diff
@@ -194,7 +194,11 @@ __device__ inline void bitonicSortKeys(
 // dimension as the innermost dim, such that we can get the particular slice for
 // a Tensor via its linear block dimension * the slice size.
 template <typename T, unsigned int Power2Size>
-__global__ void compute_mode(
+__global__ void
+#if __CUDA_ARCH__ == 620
+  __launch_bounds__(1024, 1)
```

Comment:
Runtime block size is not obvious, but 1024 is the most conservative choice, so this is probably fine.